Error initializing a new CDKTF project with the "go" template: Error: Command failed: go get

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### cdktf & Language Versions

```
$ cdktf --version
0.8.6
$ terraform --version
Terraform v1.1.3
on darwin_amd64
$ node --version
v14.18.3
$ go version
go version go1.17.5 darwin/amd64
```

### Affected Resource(s)

Initializing a new project with the `go` template, with the TFC backend (no `--local` flag)

### Debug Output

[Gist](https://gist.github.com/robin-norwood/8398a9de1ac1894d0acdc34d5cb07172)

### Expected Behavior

Successfully init project.

### Actual Behavior

```
$ cdktf init --template="go"
[...full output in gist above...]
Generating Terraform Cloud configuration for 'hashicorp-learn' organization and 'learn-cdktf-go-rln' workspace.....
# cdk.tf/go/stack
./main.go:21:17: undefined: jsii
./main.go:22:17: undefined: jsii
./main.go:23:47: undefined: jsii
Error: Command failed: go get
    at checkExecSyncError (child_process.js:790:11)
    at execSync (child_process.js:863:15)
    at Object.exports.post (/usr/local/lib/node_modules/cdktf-cli/templates/go/.hooks.sscaff.js:44:3)
    at f (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cdktf.js:116:1085)
    at imr (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cdktf.js:116:537)
    at async gGe (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cdktf.js:123:7)
    at async Object.handler (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cdktf.js:123:7789)
```

### Steps to Reproduce

In an empty directory, run cdktf init without `--local` and select the defaults.

```
% cdktf init --template="go"
```

### Important Factoids

With the `--local` flag, `cdktf init` runs fine, and adding the TFC RemoteBackend code after the fact also seems to work fine.

### References

--- a/packages/cdktf-cli/templates/go/.hooks.sscaff.js
+++ b/packages/cdktf-cli/templates/go/.hooks.sscaff.js
@@ -56,5 +56,8 @@ function terraformCloudConfig(baseName, organizationName, workspaceName) {
 		Workspaces:   cdktf.NewNamedRemoteWorkspace(jsii.String("${workspaceName}")),
 	})`);
 
+  // add import for jsii helper used above
+  result = result.replace(`"github.com/aws/constructs-go/constructs/v10"`, `"github.com/aws/constructs-go/constructs/v10"\n	"github.com/aws/jsii-runtime-go"`);
+
   writeFileSync('./main.go', result, 'utf-8');
 }

